formspree/app.py

```diff
@@ -2,18 +2,31 @@
 from flask import g
 from flask.ext.sqlalchemy import SQLAlchemy
 from flask.ext.login import LoginManager, current_user
-import fakeredis, redis
+import redis
 import settings
 
+
 DB = SQLAlchemy()
-if settings.TESTING:
-    REDIS = fakeredis.FakeStrictRedis()
-else:
-    REDIS = redis.StrictRedis.from_url(settings.REDIS_URL)
+_redis = None
+
+
+def REDIS():
+    ''' function to acquire the global redis connection '''
+    global _redis
+    if not _redis:
+        _redis = get_redis()
+    return _redis
```

Comment:
Another alternative would be to have something similar to what Django does with database, cache, file storage, have a settings for the redis backend, something like.

settings.py:
```
if testing:
  REDIS_ADAPTER = 'fakeredis.FakeStrictRedis'
else:
  REDIS_APADTER = 'redis.StrictRedis.from_url'
```

And on app.py:
```
redis = import_from_string(settings.REDIS_ADAPTER)
```

`import_from_string` being something similar to the Django one.
https://github.com/django/django/blob/master/django/utils/module_loading.py#L11